centreon/www/class/centreonStatistics.class.php

```diff
@@ -407,6 +407,48 @@ public function getAccData(): array
         return $data;
     }
 
+    /**
+     * Get poller/agent configuration data.
+     *
+     * @return array
+     */
+    public function getAgentConfigurationData(): array
+    {
+        $data = ['total' => 0];
+
+        $statement = $this->dbConfig->executeQuery(
+            <<<'SQL'
+                SELECT count(id)
+                FROM agent_configuration ac
+                SQL
+        );
+
+        $total = $statement->fetchColumn();
+
+        if ($total === false) {
+            return $data;
+        }
+
+        $data['total'] = $total;
+
+        $statement = $this->dbConfig->executeQuery(
```

Comment:
Please note that the method doesn't have the @throw tag in her php docs to alert she can throw a CentreonDbException ;)

Reply:
good !